src/map/generateStops.js

```diff
@@ -148,14 +148,7 @@ export default function (layer, timefield) {
   const rows = layer.source.data.features || layer.source.data;
   let sortedData;
   if (layer.aggregate && layer.aggregate.timeseries) {
-    sortedData = rows.sort((a, b) => {
-      if ((a.properties || a)[timefield] > (b.properties || b)[timefield]) {
-        return 1;
-      } else if ((b.properties || b)[timefield] > (a.properties || a)[timefield]) {
-        return -1;
-      }
-      return 0;
-    });
+    sortedData = rows.sort((a, b) => new Date((a.properties || a)[timefield]) - new Date((b.properties || b)[timefield]));
```

Comment:
@KipSigei Could we check if `new Date(...)` returns a valid date and use the previous sort function if it fails?

Reply:
addressed here https://github.com/onaio/gisida/pull/247/commits/755dc1e06e31a6a8af9ba084c0f7deac1cf30f40#diff-3dc2f78ef8f7b4111f5f4a21bae2e552R152

Reply:
👍